.env.example

```
@@ -17,3 +17,7 @@ SLACK_BOT_TOKEN=xoxb-your-slack-bot-token
 
 REDIS_HOST=localhost
 REDIS_PORT=6379
+
+OTEL_ENABLED=true
+OTEL_SERVICE_NAME=nest-repository-pattern
+OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318/v1/traces
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OTEL_EXPORTER_OTLP_ENDPOINT key should go before the OTEL_SERVICE_NAME key

(UnorderedKey)